flask_praetorian/constants.py

```diff
@@ -9,6 +9,17 @@
 DEFAULT_JWT_ALGORITHM = 'HS256'
 DEFAULT_JWT_ALLOWED_ALGORITHMS = ['HS256']
 DEFAULT_USER_CLASS_VALIDATION_METHOD = 'is_valid'
+DEFAULT_EMAIL_TEMPLATE = 'templates/flask_praetorian/registration_email.html'
+DEFAULT_CONFIRMATION_ENDPOINT = 'registration_confirmation'
+DEFAULT_CONFIRMATION_SENDER = 'you@whatever.com'
```

Comment:
I don't think this setting should have a default. It's best if defaults allow the app/extension to function correctly even if left as-is

Reply:
I don't follow this one (or it sounds like you are referring to a single line, can you point that if so?) ... can you elaborate?